src/extensions/surveys/surveys-utils.tsx

```diff
@@ -57,6 +57,9 @@ export const style = (appearance: SurveyAppearance | null) => {
               border-color: ${appearance?.borderColor || '#c9c6c6'};
               margin-top: 14px;
           }
+          .survey-box:has(.survey-question:empty):not(:has(.description)) textarea {
+              margin-top: 0;
+          }
```

Comment:
I only want to remove the margin in the event where there's no description (the selector is _gone_) AND there's no question (the selector is _empty_).  Otherwise, the margins should remain.

Reply:
> I'm using the :has pseudo-selector ... supported in all major browsers since 2023, but is this a risky change still?

This is fine in my opinion. It's a small styling detail that just won't work in a tiny minority of cases.